### ISSUE TITLE
Problems with training on DeepFashion2

### ISSUE THREAD

**Issue by Szakulli07** [OML-Team/open-metric-learning#451]
Hello, this is not really with your framework but mayby you can help me.

I try to train decent image retrival model on DeepFashion2. However I can not get anything at least near satisfying.
You reported results around 90% cmc1 on inshop DeepFashion so I thought mayby I can use it on DeepFashion2.
With resnet18 pretrained on image net without any fine tunning I get about 2% on validation set when considering top 10 results.

I've already tried using resnet18, resnet34, resnet50, some VITs all pretrained from image net. AdamW as optimizer with lr 1e-3, 1e-4, 1e-5. Triplet loss with margins varying from 1e-2 to 1. Batch hard mining, hard negative mining, easy positive mining and some other kinds of minings. Batch size depending of model from 400 to 1600. Using only shop images, only customer images, using both, training on only validation set.

But all that didn't get me more than 5.7% on this set.

I know I can use some more advanced techniques for pooling, rerankng etc but shouldn't I get at least decent result in this basic setup?

I would be more than grateful for any advice. This is small part of my master thesis but for now it is a lot harder than I supposed.

**Comment by AlekseySh** [comment]
Hey, @Szakulli07 !

> You reported results around 90% cmc1 on inshop DeepFashion so I thought mayby I can use it on DeepFashion2.

The DeepFashion dataset is not big enough, so the model trained on it is overfitted for this particular domain. So, I will not expect the generalization power of the model, though 2% is a weirdly low number. Also, the models from Zoo have been trained only on shop-quality images.

My advice to you is the following:
1. Try to run only the validation stage, without training. Pick one of the ViT models from Zoo. What are the results?
2. Check the existing approaches: look at [papers with code](https://paperswithcode.com/sota/clothes-landmark-detection-on-deepfashion2) website, and find more literature. Check if authors use models trained in contrastive setup and what scores they achieved.

**Comment by AlekseySh** [comment]
Closing due to inactivity, feel free to re-open it
